src/components/tracker.js

```diff
@@ -1,28 +1,55 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import { useNavigate } from 'react-router-dom';
 import { fetchTacker, setRegions } from '../Redux/tracker/tracker';
 import WorldMap from '../assets/worldmap.png';
 
-const Tacker = () => {
+const Tracker = () => {
   const dispatch = useDispatch();
+  const [searchItem, setSearchItem] = useState('');
   const trackerStore = useSelector((store) => Object.values(store.trackerReducer.tracker));
   useEffect(() => {
     if (!trackerStore.length) {
       fetchTacker(dispatch);
     }
   }, []);
   const navigate = useNavigate();
+  let alternate = true;
+  let nextLightColor = 0;
+  const Colors = (index) => {
+    if (index === nextLightColor && alternate) {
+      alternate = false;
+      nextLightColor = index + 3;
+      return 'isBlue grid-item-navlink';
+    }
+    if (index === nextLightColor && alternate === false) {
+      alternate = true;
+      nextLightColor = index + 1;
+      return 'isBlue grid-item-navlink';
+    }
+
+    return 'grid-item-navlink dark-background';
+  };
   return (
     <div>
       <div className="d-flex flex-column justify-content-center align-items-start img-class">
         <h1 className="header">Covid Tracker</h1>
```

Comment:
- [ ] Please use it as a title instead of putting it on the page.

Reply:
- Place this text also/ or in the `public\index.html` _head>title_ section:
![image](https://user-images.githubusercontent.com/12745474/142935297-5f793788-7fc0-421b-9360-31d38154e035.png)